Royalty/Patches/ThingDefs_Misc/Apparel_Royalty.xml

```diff
@@ -18,7 +18,7 @@
 		<value>
```

Comment:
As long as you are touching Apparel_Royalty, could you also change the patch for Apparel_ShirtRuffle and Apparel_VestRoyal ; the combined patch only successfully modifies Apparel_RobeRoyal while the data structure for the other two can't accept the patch.

example of patching that did work when I used it:
```
	<Operation Class="PatchOperationReplace">
		<xpath>Defs/ThingDef[defName="Apparel_RobeRoyal"]/statBases/StuffEffectMultiplierArmor</xpath>
		<value>
			<StuffEffectMultiplierArmor>1</StuffEffectMultiplierArmor>
		</value>
	</Operation>
	
	<Operation Class="PatchOperationAdd">
		<xpath>Defs/ThingDef[defName="Apparel_ShirtRuffle"]</xpath>
		<value>
		<statBases>
			<StuffEffectMultiplierArmor>1</StuffEffectMultiplierArmor>
		</statBases>
		</value>
	</Operation>
	
	<Operation Class="PatchOperationAdd">
		<xpath>Defs/ThingDef[defName="Apparel_VestRoyal"]/statBases</xpath>
		<value>
			<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
		</value>
	</Operation>
```
2 mult for vest to match corset,